react/Accordion/index.jsx

```diff
@@ -15,9 +16,9 @@ class AccordionItem extends Component {
 
     const { selected } = this.state
 
-    const removeSpaceCharacters = str => str.replace(/\s+/g, '')
+    const headingID = uniqueId('accordion_header_')
 
-    const bodyName = removeSpaceCharacters(label)
+    const contentID = uniqueId('accordion_content_')
```

Comment:
I just realized that the uniqueId is going to change at each render, it would be better to compute it at mount time and reuse it.

```
componentDidMount() {
  this.contentID = uniqueId('accordion_content')
}

render() {
   ...
   <div id={this.contentId} />
    ....
}
```

The same technique should be applied to the other PRs.